pixie/tests/test_selectors.py

```diff
@@ -69,7 +74,9 @@ def test_pyversion_selector(self):
         expected, dispatch_data = self.gen_dispatch_and_expected(dispatch_keys)
 
         selector_class = PyVersionSelector
-        dso_handler = shmEmbeddedDSOHandler()
+        # FIXME: shmEmbeddedDSOHandler probably used the wrong shm_open flag
+        # dso_handler = shmEmbeddedDSOHandler()
```

Comment:
```suggestion
```
I guess using the `mkstemp` DSO handler everywhere is fine, the test isn't about checking that, it's about checking the Python version selector.